### PR TITLE
[ARMv7] Temp fix for Android build

### PR DESCRIPTION
**Pull request by Mullin**
This fix https://code.google.com/p/dolphin-emu/issues/detail?id=8242

### CODE REVIEW

**Comment by magumagu** [comment]
This needs to be rebased.

**Comment by lioncash** [comment]
_M_ARM_32 is a define, not an actual concrete variable.

Also please don't do hack fixes like this. Fix the problem once and do it right, then you never have to care about it again.

**Comment by Mullin** [comment]
lioncash : The last build which work for Android is 4935, there is 600 build behind master. I will not have much time in the coming days. So this fix is enought to make Armv7 recompiler work again.

**Comment by Mullin** [comment]
@lioncash : See my update post above.

**Comment by lioncash** [comment]
I have seen it

File: Source/Core/Core/PowerPC/JitArm32/JitArm_LoadStore.cpp [hunk 1]
```diff
@@ -464,7 +464,7 @@ void JitArm::lXX(UGeckoInstruction inst)
 		rA = gpr.GetReg();
 
 		MOVI2R(rA, (u32)&PowerPC::OnIdle);
-		MOVI2R(R0, PowerPC::ppcState.gpr[a] + (s32)(s16)inst.SIMM_16);
+		MOVI2R(R0, PowerPC::ppcState.gpr[a] + (s32)(s16)inst.SIMM_16); // Temp Fix
```
Comment (lioncash) [inline Source/Core/Core/PowerPC/JitArm32/JitArm_LoadStore.cpp]: All this does is append `// Temp Fix` to the end of the line. Nothing else is changed.

Comment (Mullin) [inline Source/Core/Core/PowerPC/JitArm32/JitArm_LoadStore.cpp]: In master this line is removed.

File: Source/Core/Core/CoreTiming.cpp [hunk 2]
```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```
Comment (lioncash) [inline Source/Core/Core/CoreTiming.cpp]: This should most definitely not be necessary, hence why I closed this. If there's an issue in the JIT, it's in the JIT code, not the timing or event code.

Comment (Mullin) [inline Source/Core/Core/CoreTiming.cpp]: @lioncash : Without these two lines ARMv7 recompiler won't work. I cant remove the check but i don't want create a regression on other platformer although the impact will be zero. By the way i don't know if Arm64 recompiler work with Magumagu fix, there is only HdkR which use Arm64 recompiler.

Comment (lioncash) [inline Source/Core/Core/CoreTiming.cpp]: What I'm saying is that you're likely doing something that doesn't actually solve the underlying problem. All other JITs work fine with @magumagu's change. You should actually trace down *why* the JIT for ARMv7 does not work with it and fix it.

Comment (Mullin) [inline Source/Core/Core/CoreTiming.cpp]: @lioncash : I know but haven't the time right now and nobody work on ARMv7 except me and Sonicadvance(which concentrate his work on Aarch64). So in three months the problem will still be there.

Comment (lioncash) [inline Source/Core/Core/CoreTiming.cpp]: Well, we can just wait for a proper fix then.

Comment (Mullin) [inline Source/Core/Core/CoreTiming.cpp]: @lioncash : Reverse the situation, it would never happen (at least not for as long) on the JIT64

Comment (lioncash) [inline Source/Core/Core/CoreTiming.cpp]: A hacky fix on JIT64 would be equally unacceptable.

Comment (Mullin) [inline Source/Core/Core/CoreTiming.cpp]: @lioncash : That's not a hacky fix just a revert of a merged commit.

Comment (lioncash) [inline Source/Core/Core/CoreTiming.cpp]: Which further enforces the fact this revert is an incorrect fix to a problem. If two other JITs can work fine, and the outlying one does not, it's a problem with the JIT, not the merged code.

**Pull request closed without merging**